.cicdtemplate/.github/workflows/deploy_staging_and_production_to_firebase_app_distribution.yml

```diff
@@ -46,16 +46,15 @@ jobs:
           name: DetektReports
           path: build/reports/detekt/
 
-      - name: Run unit tests and Jacoco
-        run: ./gradlew jacocoTestReport
+      - name: Run unit tests with Kover
+        run: ./gradlew koverMergedHtmlReport
```

Comment:
Should we be calling `koverMergedReport` instead?

Then we can archive the HTML version, and still use XML version for Danger 👯

Reply:
@ryan-conway My bad... This one is correct already 🙇  

But for `run_detekt_and_unit_tests.yml` it should be `koverMergedHtmlReport` if I'm not mistaken.